ui/app/templates/release_scheduled_changes.html

```diff
@@ -69,7 +69,7 @@ <h3 class="panel-title">
     </h3>
   </div>
   <div class="panel-body">
-    <div ng-show="!sc.sc_id && ! isEmpty(sc.required_signoffs)">
+    <div ng-show="!sc_id && ! isEmpty(sc.required_signoffs)">
       <span style="float: left">
         <h4>Before it can be enacted, this change must be signed off on by:</h4>
         <ul>
```

Comment:
I don't think so. On line 65, we're adding a link from the Scheduled Changes page to the Scheduled Changes History page -- which means we want the sc_id of the current object from the loop. Similar for line 100, we're displaying the sc_id of the current object. This code is shared by both the Scheduled Changes and the History page, so we need to use the iterated one because it differs between items on the former.